spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -444,7 +444,7 @@ def main(argv=None):
     # Normalize CSA values (MEAN(area))
     if arguments.normalize is not None:
         data_subject = pd.DataFrame([arguments.normalize])
-        path_model = os.path.join(__sct_dir__, 'spinalcordtoolbox', 'data', 'csa_normalization_models',
+        path_model = os.path.join(__sct_dir__, 'data', 'csa_normalization_models',
                                   '_'.join(sorted(data_subject.columns)) + '.csv')
```

Comment:
Ahhhhh, I see why the tests are failing. It's because of this line in `install_sct`:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/ae312ed9d685b1a9dffc8284925317cbf2893a8e/install_sct#L675

Currently, we wipe the existing `data/` file before re-installing the data. So, storing files in `data/` by default will just have those files deleted on install. 

I missed this when I first proposed my original suggestion in #3558. :sweat:

Reply:
Truthfully, I think that the `rm -rf` step might not even be necessary?

`sct_download_data` _already_ removes subfolders for individual `data/` downloads:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/ae312ed9d685b1a9dffc8284925317cbf2893a8e/spinalcordtoolbox/download.py#L140-L142

So, it feels a bit redundant to start from an empty data folder in `install_sct`. That just feels overly destructive to me, as it will clear out _everything_ in `data/`, including any extra files that users have placed there.

I'd propose simply removing the `rm -rf` line entirely from `install_sct`.

Reply:
Could you please delete L675 from `install_sct`, @jqmcginnis? 

I believe this should make the failing tests pass. :)

Reply:
Thank you very much for debugging this! Should be resolved in [50eee3d](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3821/commits/50eee3d1fbf4b43369d1def31da72f694bab5885) :slightly_smiling_face: !